class/cert-manager.yml

```
@@ -34,10 +65,15 @@ parameters:
           name: cert-manager
           namespace: ${cert_manager:namespace}
         output_path: cert-manager/10_cert_manager
+      - ${_config:exoscale_webhook_values:${cert_manager:components:exoscale_webhook:enabled}}
+      - ${_config:exoscale_webhook_helm:${cert_manager:components:exoscale_webhook:enabled}}
 
   commodore:
     postprocess:
       filters:
         - type: jsonnet
           path: cert-manager/10_cert_manager/cert-manager/templates/
           filter: postprocess/patch-crds.jsonnet
+        - type: jsonnet
+          path: cert-manager/20_exoscale_webhook/exoscale-webhook/templates/
+          filter: postprocess/patch-exoscale.jsonnet
```

afaict this should have

```suggestion
          filter: postprocess/patch-exoscale.jsonnet
          enabled: ${cert_manager:components:exoscale_webhook:enabled}
```

---

Ah I was not aware of such a switch, fantastic.
I handle that in the jsonnet, but that is nicer.